src/shared-ui/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@niis/shared-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "build-lib": "vue-cli-service build --target lib --name shared-ui ./src/lib.ts",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "main": "./dist/shared-ui.common.js",
+  "types": "src/lib.d.ts",
+  "dependencies": {
+    "@mdi/font": "~5.6.55",
+    "core-js": "^3.6.5",
+    "roboto-fontface": "*",
+    "vue": "^2.6.11",
+    "vue-router": "^3.2.0",
+    "vuetify": "^2.2.11",
+    "vuex": "^3.4.0",
+    "vue-i18n": "~8.21.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.6",
+    "@storybook/addon-actions": "^6.0.26",
+    "@storybook/addon-essentials": "^6.0.26",
+    "@storybook/addon-links": "^6.0.26",
+    "@storybook/vue": "^6.0.26",
+    "@types/jest": "~26.0.8",
+    "@typescript-eslint/eslint-plugin": "~4.6.0",
+    "@typescript-eslint/parser": "~4.6.0",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-eslint": "~4.5.0",
+    "@vue/cli-plugin-router": "~4.5.0",
+    "@vue/cli-plugin-typescript": "~4.5.0",
+    "@vue/cli-plugin-unit-jest": "~4.5.0",
+    "@vue/cli-plugin-vuex": "~4.5.0",
+    "@vue/cli-service": "~4.5.0",
+    "@vue/eslint-config-prettier": "^6.0.0",
+    "@vue/eslint-config-typescript": "~7.0.0",
+    "@vue/test-utils": "^1.0.3",
+    "babel-loader": "^8.1.0",
+    "eslint": "~7.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-vue": "~7.0.1",
+    "prettier": "~2.1.1",
+    "react-is": "^16.13.1",
+    "sass": "~1.26.10",
+    "sass-loader": "~10.0.1",
+    "typescript": "4.0.3",
+    "vee-validate": "3.4.1",
+    "vue-cli-plugin-vuetify": "~2.0.7",
+    "vue-cli-plugin-webpack-bundle-analyzer": "~2.0.0",
+    "vue-template-compiler": "^2.6.11",
+    "vuetify-loader": "^1.3.0",
+    "webpack-node-externals": "^2.5.2"
+  },
+  "peerDependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^2.6.11",
+    "vuetify": "^2.2.11",
+    "vue-i18n": "^8.21.0",
+    "vee-validate": "^3.4.1"
```

Comment:
I doubt these dependencies should be both in dependencies and peerDependencies?

Reply:
They need to be in both because peerDependencies are not installed. Those are demands for the project that includes the library.

Reply:
I did some more investigation and got the meaning of peerDependencies better now. So this is ok.